cyhy_commander/commander.py

```diff
@@ -617,6 +703,36 @@ def do_work(self):
         self.__setup_sources()
         self.__setup_sinks()
 
+        self.__successful_job_queue = Queue.Queue()
+        self.__failed_job_queue = Queue.Queue()
+
+        # spin up the thread pool to process retrieved work
+        job_processing_threads = []
+        for t in range(job_processing_thread_count):
+            job_processing_thread = threading.Thread(target=self.__process_queued_jobs)
+            job_processing_threads.append(job_processing_thread)
+            try:
+                job_processing_thread.start()
+            except Exception as e:
+                self.__logger.error("Unable to start job processing thread #%s", t)
+                self.__logger.error(e)
+                # bail out
+                self.__logger.critical(
+                    "Shutting down due to inability to start threads."
+                )
+                self.__is_running = False
+
+        # spin up a thread to output queue load information
+        job_queue_monitor_thread = threading.Thread(target=self.__monitor_job_queues)
+        try:
+            job_queue_monitor_thread.start()
+        except Exception as e:
+            self.__logger.error("Unable to start job queue monitoring thread")
+            self.__logger.error(e)
+            # bail out
+            self.__logger.critical("Shutting down due to inability to start threads.")
```

Comment:
Now that we have different types of threads, we can customize the failure messages:
```suggestion
            self.__logger.critical("Shutting down due to inability to start queue monitoring thread.")
```